lua/entities/gmod_wire_expression2/core/custom/sv_selfaware2.lua

```diff
@@ -1,54 +1,125 @@
 -- Author: Vurv, define runtime functions
 
 --[[
-   _____        __ ____   ___                                        ___ 
-  / ___/ ___   / // __/  /   | _      __ ____ _ _____ _____ ___     |__ \
-  \__ \ / _ \ / // /_   / /| || | /| / // __ `// ___// ___// _ \    __/ /
- ___/ //  __// // __/  / ___ || |/ |/ // /_/ // /   / /   /  __/   / __/ 
-/____/ \___//_//_/    /_/  |_||__/|__/ \__,_//_/   /_/    \___/   /____/ 
+   _____        __ ____   ___                                  ___
+  / ___/ ___   / // __/  /   | _      __ ____ _ _____ ___     |__ \
+  \__ \ / _ \ / // /_   / /| || | /| / // __ `// ___// _ \    __/ /
+ ___/ //  __// // __/  / ___ || |/ |/ // /_/ // /   /  __/   / __/
+/____/ \___//_//_/    /_/  |_||__/|__/ \__,_//_/    \___/   /____/
 
  Adds functions similarly to regular-e2's self-aware core.
 ]]
 
--- Reused from coroutine-core
-local function getE2UDF(compiler,funcname)
```

Comment:
Same as earlier, this has been refactored and moved into the helpers.
(Same for the `getE2Func` below.)
We import these functions here so we can use them (line 15).